src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+    >
+      <fieldset class="first">
+        <legend>Personal information</legend>
+
+        <label class="personal_info first">
+          Surname:
+          <input 
+          type="text" 
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="surname"
            autocomplete="off"
          >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

fix it all over the page